src/System.IO.Ports/tests/Legacy/SerialStream/BeginRead.cs

```diff
@@ -1,788 +1,539 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
+//  Licensed to the .NET Foundation under one or more agreements.
+//  The .NET Foundation licenses this file to you under the MIT license.
+//  See the LICENSE file in the project root for more information.
```

Comment:
Doh, sorry - I applied a regex across the whole thing to deal with comments which were like `//Comment` and didn't notice it had done that.